docker/docker-compose.yml

```diff
@@ -31,9 +32,11 @@ services:
         image: aegee/oms-mail-transfer-agent:latest
         expose:
             - "25"
-        secrets:
-            - mail_password
-            - mail_user
+        environment:
+            SMTP_HOST: ${SMTP_HOST}
+            SMTP_PORT: 587
+            SMTP_USER: ${SMTP_USER}
```

Comment:
Perhaps you could make the variables mandatory like you showed on slack

Reply:
done

Reply:
fuck, the tests are broken now. need to fix travis/circle ci, then it's done